content/zh/docs/reference/kubernetes-api/workload-resources/cron-job-v1.md

```diff
@@ -0,0 +1,1559 @@
+
+<!--
+## CronJob {#CronJob}
+
+CronJob represents the configuration of a single cron job.
+-->
+
+
+<hr>
+
+- **apiVersion**: batch/v1
+
+<!--
+<hr>
+
+- **apiVersion**: batch/v1
+-->
+
+- **kind**: CronJob
+
+<!-- - **kind**: CronJob-->
+
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  标准对象的元数据。
+  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  
+<!--
+- **metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+  Standard object's metadata. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+-->  
+
+- **spec** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobSpec" >}}">CronJobSpec</a>)
+
+  指定 cron 作业所期望的行为，包含明细表。
+  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+<!--
+- **spec** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobSpec" >}}">CronJobSpec</a>)
+
+  Specification of the desired behavior of a cron job, including the schedule. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+
+- **status** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobStatus" >}}">CronJobStatus</a>)
+  cron 作业的当前状态。
+  更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+
+<!--
+- **status** (<a href="{{< ref "../workload-resources/cron-job-v1#CronJobStatus" >}}">CronJobStatus</a>)
+
+  Current status of a cron job. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+-->
+
+
+
+
+## CronJobSpec {#CronJobSpec}
+
+CronJobSpec 描述了作业执行的样子以及实际运行的时间。
+
+<hr>
+
+<!--
+## CronJobSpec {#CronJobSpec}
+
+CronJobSpec describes how the job execution will look like and when it will actually run.
+
+<hr>
+-->
+
+- **jobTemplate** (JobTemplateSpec)，必填
+
+  指定执行 CronJob 时将创建的作业。
+  
+  <a name="JobTemplateSpec"></a>
+  *JobTemplateSpec 描述了从模板创建作业时应具有的数据。*
+  
+<!--
+- **jobTemplate** (JobTemplateSpec), required
+
+  Specifies the job that will be created when executing a CronJob.
+  
+   <a name="JobTemplateSpec"></a>
+   *JobTemplateSpec describes the data a Job should have when created from a template*
+-->
+
+  - **jobTemplate.metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+
+    从此模板创建的作业的标准对象元数据。
+    更多信息：https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+    
+  <!--
+  - **jobTemplate.metadata** (<a href="{{< ref "../common-definitions/object-meta#ObjectMeta" >}}">ObjectMeta</a>)
+  
+      Standard object's metadata of the jobs created from this template. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+  -->
+
+  - **jobTemplate.spec**
```

Comment:
This line is different from the English page.

```suggestion
  - **jobTemplate.spec** (<a href="{{< ref "../workload-resources/job-v1#JobSpec" >}}">JobSpec</a>)
```

Reply:
ok, i will delete this href temporarily

Reply:
so i delete the ref, but now the checks also failed.

Reply:
although i delete the href,  is it still has error?